src/content/docs/usage/builders.mdx

```diff
@@ -5,6 +5,12 @@ sidebar:
   label: Builders
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
```

Comment:
Builders are only available in Open Source

Reply:
> @Tpuljak there is a devcontiner builder in EE?

@ivan-burazin you can create a devcontainer in EE but it's not really a builder. Also, there's no custom image builder.